.github/workflows/ci.yml

```diff
@@ -194,6 +194,12 @@ jobs:
       - uses: actions/checkout@v4
       - uses: Swatinem/rust-cache@v2
       - name: install cargo-msrv
-        run: cargo install cargo-msrv
-      - name: check msrv
-        run: cargo msrv --output-format json verify -- cargo check
\ No newline at end of file
+        run: cargo install --git https://github.com/foresterre/cargo-msrv.git cargo-msrv
+      - name: check msrv for trippy
+        run: cargo msrv verify --output-format json --manifest-path crates/trippy/Cargo.toml -- cargo check
+      - name: check msrv for trippy-core
+        run: cargo msrv verify --output-format json --manifest-path crates/trippy-core/Cargo.toml -- cargo check
+      - name: check msrv for trippy-dns
+        run: cargo msrv verify --output-format json --manifest-path crates/trippy-dns/Cargo.toml -- cargo check
+      - name: check msrv for trippy-privilege
+        run: cargo msrv verify --output-format json --manifest-path crates/trippy-privilege/Cargo.toml -- cargo check
```

Comment:
Switched to master for cargo-msrv to get (partial) support for virtual workspaces, but requires it be run once per crate.